### PR TITLE
[CALCITE-6105] Documentation does not specify the behavior of SPLIT

### PR DESCRIPTION
**Pull request by mihaibudiu**
function for empty string arguments

### CODE REVIEW

File: site/_docs/reference.md [hunk 1]
```diff
@@ -2823,7 +2823,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b m o p | SOUNDEX(string)                          | Returns the phonetic representation of *string*; throws if *string* is encoded with multi-byte encoding such as UTF-8
 | s | SOUNDEX(string)                                | Returns the phonetic representation of *string*; return original *string* if *string* is encoded with multi-byte encoding such as UTF-8
 | m | SPACE(integer)                                 | Returns a string of *integer* spaces; returns an empty string if *integer* is less than 1
-| b | SPLIT(string [, delimiter ])                   | Returns the string array of *string* split at *delimiter* (if omitted, default is comma)
+| b | SPLIT(string [, delimiter ])                   | Returns the string array of *string* split at *delimiter* (if omitted, default is comma).  If the string is empty it returns an empty vector, otherwise, if the delimiter is empty, it returns a vector containing the original string.
```
[the remark this answers is not in the document]
Comment (mihaibudiu) [reply site/_docs/reference.md]: Sorry for the omission. I have squashed the commits optimistically.

[the remark this answers is not in the document]
Comment (tanclary) [reply site/_docs/reference.md]: Sorry for the nit but should we remove `it returns a vector containing the original string` in favor of `if the delimiter is empty, returns *string*`?

[the remark this answers is not in the document]
Comment (mihaibudiu) [reply site/_docs/reference.md]: I forgot to quote this one. But it always returns a vector.

[the remark this answers is not in the document]
Comment (mihaibudiu) [reply site/_docs/reference.md]: Changed to "array" instead of "vector", that's the terminology used everywhere.

**Review by tanclary — APPROVED** [review]
LGTM

**Pull request merged**